key.go

```diff
@@ -5,55 +5,65 @@ package crypto
 
 import (
 	"bytes"
-	"encoding/base64"
-	"errors"
-	"fmt"
-	"io"
-
 	"crypto/elliptic"
 	"crypto/hmac"
 	"crypto/rand"
-	"crypto/rsa"
 	"crypto/sha1"
 	"crypto/sha512"
+	"encoding/base64"
+	"errors"
+	"fmt"
 	"hash"
-
-	pb "github.com/libp2p/go-libp2p-crypto/pb"
+	"io"
 
 	"github.com/gogo/protobuf/proto"
+	pb "github.com/libp2p/go-libp2p-crypto/pb"
```

Comment:
FYI, we usually order our imports:

1. stdlib
2. local repo
3. external repos

(with newlines in-between)

However, the separation between the crypto and non-crypto imports was *non*-idiomatic.